tests/main/interfaces-desktop-launch/task.yaml

```diff
@@ -0,0 +1,56 @@
+summary: Ensure that the desktop-launch interface works.
+
+details: |
+    The desktop-launch interface allows a snap to launch other snaps via
+    the desktop files they provide to the host system.
+
+systems: [-ubuntu-core-*]
```

Comment:
could you add a comment that userd does not yet work properly enough on ubuntu core to be able to run this test there (fine for followup)?